src/openforms/forms/api/viewsets.py

```diff
@@ -211,7 +212,7 @@ class FormViewSet(viewsets.ModelViewSet):
     lookup_url_kwarg = "uuid_or_slug"
     # lookup_value_regex = "[0-9a-fA-F]{8}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{12}"
     serializer_class = FormSerializer
-    permission_classes = [IsStaffOrReadOnly]
+    permission_classes = [IsStaffOrReadOnlyNoList]
```

Comment:
Be advised that this may break potential integration from CMS solutions which relied on this endpoint being public.

We will have to check/set-up token auth for this and document it.